README.md

```diff
@@ -77,4 +77,13 @@ REMS is an open source project. In case you would like to contribute to its deve
 
 ## More documentation
 
-See the [docs](./docs) folder
+Documentation files can be found under the [docs](./docs) folder.
+
+Documentation can also be read from the browser by launching user docs server with the command:
+`mkdocs serve`
```

Comment:
installing mkdocs and ...?

Reply:
Well let's point people to use dockerfile ;)